src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage.mdx

```diff
@@ -171,5 +202,383 @@ let poolSizeSet = NewRelic.setMaxOfflineStorageSize(1000)
 
         </TabsPageItem>
 
+
+ <TabsPageItem id="capacitor">
+
+ <Callout variant="important">
+Offline storage is enabled by default but can be disabled with a feature flag using the agent configuration.
+</Callout>
+
+## Syntax [#syntax]
+
+```typescript
+setMaxOfflineStorageSize(options: { megabytes: number; }) => void
+```
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width={200}>
+        Parameter
+      </th>
+
+            <th width={200}>
+        Type
+      </th>
+
+      <th>
+        Description
+      </th>
+
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `megabytes`
+      </td>
+
+      <td>
+        `number`
+      </td>
+
+      <td>
+         Required. Maximum size in megaBytes that can be stored in the file system.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Example [#example]
+
+```typescript
+        NewRelicCapacitorPlugin.setMaxOfflineStorageSize({ megaBytes: 200 })
+```
+
+        </TabsPageItem>
+        <TabsPageItem id="cordova">
+
+   <Callout variant="important">
+Offline storage is enabled by default but can be disabled with a feature flag using the agent configuration.
```

Comment:
@ndesai-newrelic Can you add the code sample of how to disable this (similar to the iOS/Android callouts)?